compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -465,6 +465,37 @@ fn trait_constraint_eq(
     Ok(Value::Bool(constraint_a == constraint_b))
 }
 
+// fn trait_def_hash(def: TraitDefinition) -> Field
+fn trait_def_hash(
+    _interner: &mut NodeInterner,
+    mut arguments: Vec<(Value, Location)>,
+    location: Location,
+) -> IResult<Value> {
+    check_argument_count(1, &arguments, location)?;
+
+    let id = get_trait_def(arguments.pop().unwrap().0, location)?;
```

Comment:
Do we need to document whether this is stable across compilations, versions, etc?

Reply:
No, I'm including this as part of the experimental comptime work which is still undocumented.
There'll be a lot of documentation required once we stabilize it all.